ccm_web/client/src/components/InlineTextEdit.tsx

```diff
@@ -78,20 +83,29 @@ function InlineTextEdit (props: InlineTextEditProps): JSX.Element {
     }
   }
 
+  const renderEditButton = (): JSX.Element => {
+    if (!props.isSaving) {
+      return (<EditIcon className={classes.editIcon} fontSize='inherit' onClick={toggleEdit}/>)
+    } else {
+      return (<EditIcon className={classes.editIconDisabled} fontSize='inherit'/>)
+    }
+  }
+
   return (
     <form className={classes.root} noValidate autoComplete="off">
+      Is Saving &quot;{props.isSaving.toString()}&quot;
```

Comment:
<img width="379" alt="Screen Shot 2021-07-07 at 3 59 24 PM" src="https://user-images.githubusercontent.com/8579775/124821170-61656980-df3c-11eb-9da2-93257d19d1e0.png">

Something here is messing it up in the UI